*** 2017-08-13 16:00:52.232 idempiere Log (CLogFile) ***
16:00:52.231 Adempiere.startup: iDempiere(r) Release 3.1_2015-10-31 -Smart Suite ERP, CRM and SCM- (c) 1999-2013 iDempiere(r); Implementation: iDempiere - Supported by iDempiere community [17]
16:00:52.235 Ini.loadProperties: E:\mercurialidempiere/idempiere.properties #33 [17]
16:00:53.113 AdempiereActivator.installPackage: ni.idempiere.importcurrencyrate 1.0.0 was installed: 2017-05-19 12:08:41.274 [17]
16:00:53.116 AdempiereActivator.installPackage: cm.itkamer.wandapos-integration2 1.0.0 was installed: 2017-07-10 23:50:08.486 [17]
16:00:53.120 AdempiereActivator.installPackage: ni.idempiere.override.codebase 1.0.0 was installed: 2017-08-07 00:33:04.411 [17]
16:00:53.133 DB.isBuildOK: Build DB=${env.ADEMPIERE_VERSION} 20080428-1232 [17]
16:00:53.133 DB.isBuildOK: Build Cl=3.1.0.qualifier [17]
16:00:53.187-----------> DB.isBuildOK: Error en la versión de la construcción 

El programa asume version de construcción 3.1.0.qualifier, pero la base de datos tiene una versión ${env.ADEMPIERE_VERSION} 20080428-1232.
Es posible que esto cause errores difíciles de corregir.
Por favor contacte al ADMINISTRADOR. [17]
16:00:53.372 WebEnv.initWeb: Servlet Context Init Parameters: iDempiere Root
ServerRoot=1 [17]
16:00:53.591-----------> EMail.setFrom: you @ company.org: javax.mail.internet.AddressException: Local address contains control or whitespace in string ``you @ company.org'' [17]
16:00:53.593-----------> EMail.addTo: you @ company.org: javax.mail.internet.AddressException: Local address contains control or whitespace in string ``you @ company.org'' [17]
16:00:53.594-----------> EMail.isValid: From is invalid=null [17]
16:00:53.594-----------> EMail.isValid: From is invalid=null [17]
16:00:53.594-----------> MClient.sendEMail: Could NOT Send Email: Server started: iDempiere (odbz:192.168.47.1) to you @ company.org: Invalid Data (System) [17]
16:00:53.594 WebEnv.initWeb: Servlet Init Parameter: idempiereMonitor [17]
16:00:53.595 AdempiereMonitor.init:  [17]
16:00:56.432-----------> AnnotationConfiguration.configure: ServletContainerInitializers: detected. Class hierarchy: empty [17]
16:00:56.715-----------> ClassLocator.getDependentXMLResources: Replicate resource: ckez
Overwrite bundleresource://188.fwk47108597:2/metainfo/zk/config.xml
with bundleresource://188.fwk47108597:19/metainfo/zk/config.xml [17]
16:00:56.724-----------> ClassLocator.getDependentXMLResources: Replicate resource: zk
Overwrite bundleresource://188.fwk47108597:11/metainfo/zk/config.xml
with bundleresource://188.fwk47108597:28/metainfo/zk/config.xml [17]
16:00:56.730-----------> ClassLocator.getDependentXMLResources: Replicate resource: zkbind
Overwrite bundleresource://188.fwk47108597:12/metainfo/zk/config.xml
with bundleresource://188.fwk47108597:29/metainfo/zk/config.xml [17]
16:00:56.761-----------> ClassLocator.getDependentXMLResources: Replicate resource: zul
Overwrite bundleresource://188.fwk47108597:14/metainfo/zk/zk.xml
with bundleresource://188.fwk47108597:31/metainfo/zk/zk.xml [17]
16:00:57.158-----------> LanguageDefinition.<init>: Replicated language: xhtml, overriden by [LanguageDefinition: xhtml] [17]
16:00:57.180-----------> LanguageDefinition.<init>: Replicated language: xul/html, overriden by [LanguageDefinition: xul/html] [17]
16:00:57.238-----------> ClassLocator.getDependentXMLResources: Replicate resource: calendar
Overwrite bundleresource://188.fwk47108597:1/metainfo/zk/lang-addon.xml
with bundleresource://188.fwk47108597:18/metainfo/zk/lang-addon.xml [17]
16:00:57.242-----------> ClassLocator.getDependentXMLResources: Replicate resource: ckez
Overwrite bundleresource://188.fwk47108597:2/metainfo/zk/lang-addon.xml
with bundleresource://188.fwk47108597:19/metainfo/zk/lang-addon.xml [17]
16:00:57.248-----------> ClassLocator.getDependentXMLResources: Replicate resource: gmapsz
Overwrite bundleresource://188.fwk47108597:5/metainfo/zk/lang-addon.xml
with bundleresource://188.fwk47108597:22/metainfo/zk/lang-addon.xml [17]
16:00:57.251-----------> ClassLocator.getDependentXMLResources: Replicate resource: timeline
Overwrite bundleresource://188.fwk47108597:6/metainfo/zk/lang-addon.xml
with bundleresource://188.fwk47108597:23/metainfo/zk/lang-addon.xml [17]
16:00:57.256-----------> ClassLocator.getDependentXMLResources: Replicate resource: timeplot
Overwrite bundleresource://188.fwk47108597:7/metainfo/zk/lang-addon.xml
with bundleresource://188.fwk47108597:24/metainfo/zk/lang-addon.xml [17]
16:00:57.272-----------> ClassLocator.getDependentXMLResources: Replicate resource: zkbind
Overwrite bundleresource://188.fwk47108597:12/metainfo/zk/lang-addon.xml
with bundleresource://188.fwk47108597:29/metainfo/zk/lang-addon.xml [17]
16:00:57.281-----------> ClassLocator.getDependentXMLResources: Replicate resource: zkplus
Overwrite bundleresource://188.fwk47108597:13/metainfo/zk/lang-addon.xml
with bundleresource://188.fwk47108597:30/metainfo/zk/lang-addon.xml [17]
16:00:57.602           WebUIServlet.init: iDempiere Web Client started successfully [17]
16:00:58.718-----------> AnnotationConfiguration.configure: ServletContainerInitializers: detected. Class hierarchy: empty [17]

*** 2017-08-13 16:00:58.972 idempiere Log (CLogFile) ***
16:01:01.237-----------> Node: [192.168.163.1]:5701 [adempiere.idempiere/localhost] [3.5.3] No join method is enabled! Starting standalone. [41]
16:02:11.943===========> MOrder.load: NO Data found for C_Order_ID=1005073 [37]
java.lang.Exception
	at org.compiere.model.PO.load(PO.java:1347)
	at org.compiere.model.PO.load(PO.java:1287)
	at org.compiere.model.PO.<init>(PO.java:204)
	at org.compiere.model.PO.<init>(PO.java:154)
	at org.compiere.model.X_C_Order.<init>(X_C_Order.java:41)
	at org.compiere.model.MOrder.<init>(MOrder.java:157)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.adempiere.base.DefaultModelFactory.getPO(DefaultModelFactory.java:272)
	at org.compiere.model.MTable.getPO(MTable.java:397)
	at org.compiere.model.MRecentItem.getLabel(MRecentItem.java:289)
	at org.adempiere.webui.dashboard.DPRecentItems.refresh(DPRecentItems.java:236)
	at org.adempiere.webui.dashboard.DPRecentItems.createRecentItemsPanel(DPRecentItems.java:145)
	at org.adempiere.webui.dashboard.DPRecentItems.<init>(DPRecentItems.java:98)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at java.lang.Class.newInstance(Unknown Source)

16:02:12.067===========> GenericPO.load: NO Data found for C_Order_ID=1005073 [37]
java.lang.Exception
	at org.compiere.model.PO.load(PO.java:1347)
	at org.compiere.model.PO.load(PO.java:1287)
	at org.compiere.model.PO.<init>(PO.java:204)
	at org.adempiere.model.GenericPO.<init>(GenericPO.java:71)
	at org.compiere.model.MTable.getPO(MTable.java:410)
	at org.compiere.model.MRecentItem.getLabel(MRecentItem.java:289)
	at org.adempiere.webui.dashboard.DPRecentItems.refresh(DPRecentItems.java:236)
	at org.adempiere.webui.dashboard.DPRecentItems.createRecentItemsPanel(DPRecentItems.java:145)
	at org.adempiere.webui.dashboard.DPRecentItems.<init>(DPRecentItems.java:98)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at java.lang.Class.newInstance(Unknown Source)
	at org.zkoss.zk.ui.metainfo.impl.ComponentDefinitionImpl.newInstance(ComponentDefinitionImpl.java:312)
	at org.zkoss.zk.ui.metainfo.impl.ComponentDefinitionImpl.newInstance(ComponentDefinitionImpl.java:299)
	at org.zkoss.zk.ui.metainfo.ComponentInfo.newInstance(ComponentInfo.java:624)
	at org.zkoss.zk.ui.impl.AbstractUiFactory.newComponent(AbstractUiFactory.java:96)
	at org.zkoss.zk.ui.impl.UiEngineImpl.execCreateChild0(UiEngineImpl.java:857)
	at org.zkoss.zk.ui.impl.UiEngineImpl.execCreateChild(UiEngineImpl.java:826)
	at org.zkoss.zk.ui.impl.UiEngineImpl.execCreate0(UiEngineImpl.java:735)

16:04:21.951===========> DB.executeUpdate: UPDATE I_Order SET M_Warehouse_ID=(SELECT M_Warehouse_ID FROM M_Warehouse w WHERE w.Name=I_Order.LocatorName AND I_Order.AD_Client_ID=w.AD_Client_ID), AD_Org_ID=(SELECT AD_Org_ID FROM M_Warehouse w WHERE w.Name=I_Order.LocatorName AND I_Order.AD_Client_ID=w.AD_Client_ID)WHERE M_Warehouse_ID IS NULL AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: column i_order.locatorname does not exist
  Position: 90; State=42703; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:278)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.096===========> DB.saveError: DBExecuteError - ERROR: column i_order.locatorname does not exist
  Position: 90 [186]
16:04:22.099-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.100-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:286)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.100===========> DB.executeUpdate: UPDATE I_Order SET M_Warehouse_ID=(SELECT MAX(M_Warehouse_ID) FROM M_Warehouse w WHERE I_Order.AD_Client_ID=w.AD_Client_ID AND I_Order.AD_Org_ID=w.AD_Org_ID) WHERE M_Warehouse_ID IS NULL AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:286)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.173===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.176-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.177-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:295)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.177===========> DB.executeUpdate: UPDATE I_Order SET M_Warehouse_ID=(SELECT M_Warehouse_ID FROM M_Warehouse w WHERE I_Order.AD_Client_ID=w.AD_Client_ID) WHERE M_Warehouse_ID IS NULL AND EXISTS (SELECT AD_Client_ID FROM M_Warehouse w WHERE w.AD_Client_ID=I_Order.AD_Client_ID GROUP BY AD_Client_ID HAVING COUNT(*)=1) AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:295)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.212===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.214-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.215-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:303)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.216===========> DB.executeUpdate: UPDATE I_Order SET I_IsImported='E', I_ErrorMsg=I_ErrorMsg||'ERR=No Warehouse, ' WHERE M_Warehouse_ID IS NULL AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:303)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.254===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.255-----------> NIImportOrder.doIt: No Warehouse=-1 [186]
16:04:22.257-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.258-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:313)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.258===========> DB.executeUpdate: UPDATE I_Order SET C_BPartner_ID=u.C_BPartner_ID,AD_User_ID=u.AD_User_ID FROM AD_User u WHERE I_Order.EMail=u.EMail AND I_Order.AD_Client_ID=u.AD_Client_ID AND u.C_BPartner_ID IS NOT NULL AND I_Order.C_BPartner_ID IS NULL AND I_Order.EMail IS NOT NULL AND I_Order.I_IsImported<>'Y' AND I_Order.AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:313)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.298===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.301-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.302-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:322)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.303===========> DB.executeUpdate: UPDATE I_Order SET C_BPartner_ID=u.C_BPartner_ID,AD_User_ID=u.AD_User_ID FROM AD_User u WHERE I_Order.ContactName=u.Name AND I_Order.AD_Client_ID=u.AD_Client_ID AND u.C_BPartner_ID IS NOT NULL AND I_Order.C_BPartner_ID IS NULL AND I_Order.ContactName IS NOT NULL AND EXISTS (SELECT Name FROM AD_User u WHERE I_Order.ContactName=u.Name AND I_Order.AD_Client_ID=u.AD_Client_ID AND u.C_BPartner_ID IS NOT NULL GROUP BY Name HAVING COUNT(*)=1) AND I_Order.I_IsImported<>'Y' AND I_Order.AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:322)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.316===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.319-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.320-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:330)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.321===========> DB.executeUpdate: UPDATE I_Order SET C_BPartner_ID=(SELECT MAX(C_BPartner_ID) FROM C_BPartner bp WHERE I_Order.BPartnerValue=bp.Value AND I_Order.AD_Client_ID=bp.AD_Client_ID) WHERE C_BPartner_ID IS NULL AND BPartnerValue IS NOT NULL AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:330)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.350===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.354-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.354-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:338)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.356===========> DB.executeUpdate: UPDATE I_Order SET C_BPartner_ID=(SELECT C_BPartnerCashTrx_ID FROM AD_ClientInfo c WHERE I_Order.AD_Client_ID=c.AD_Client_ID) WHERE C_BPartner_ID IS NULL AND BPartnerValue IS NULL AND Name IS NULL AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:338)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.392===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.397-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.397-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:351)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.398===========> DB.executeUpdate: UPDATE I_Order SET BillTo_ID=bpl.C_BPartner_Location_ID,C_BPartner_Location_ID=bpl.C_BPartner_Location_ID FROM C_BPartner_Location bpl INNER JOIN C_Location l ON (bpl.C_Location_ID=l.C_Location_ID) WHERE I_Order.C_BPartner_ID=bpl.C_BPartner_ID AND bpl.AD_Client_ID=I_Order.AD_Client_ID AND MD5(I_Order.Address1)=MD5(l.Address1) AND MD5(I_Order.Address2)=MD5(l.Address2) AND MD5(I_Order.City)=MD5(l.City) AND MD5(I_Order.Postal)=MD5(l.Postal) AND I_Order.C_Region_ID=l.C_Region_ID AND I_Order.C_Country_ID=l.C_Country_ID AND I_Order.C_BPartner_ID IS NOT NULL AND I_Order.C_BPartner_Location_ID IS NULL AND I_Order.I_IsImported='N' AND I_Order.AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:351)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.425===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.430-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.431-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:361)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.432===========> DB.executeUpdate: UPDATE I_Order SET BillTo_ID=(SELECT MAX(C_BPartner_Location_ID) FROM C_BPartner_Location l WHERE l.C_BPartner_ID=I_Order.C_BPartner_ID AND I_Order.AD_Client_ID=l.AD_Client_ID AND ((l.IsBillTo='Y' AND I_Order.IsSOTrx='Y') OR (l.IsPayFrom='Y' AND I_Order.IsSOTrx='N'))) WHERE C_BPartner_ID IS NOT NULL AND BillTo_ID IS NULL AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:361)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.450===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.455-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.455-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:371)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.456===========> DB.executeUpdate: UPDATE I_Order SET C_BPartner_Location_ID=(SELECT MAX(C_BPartner_Location_ID) FROM C_BPartner_Location l WHERE l.C_BPartner_ID=I_Order.C_BPartner_ID AND I_Order.AD_Client_ID=l.AD_Client_ID AND ((l.IsShipTo='Y' AND I_Order.IsSOTrx='Y') OR I_Order.IsSOTrx='N')) WHERE C_BPartner_ID IS NOT NULL AND C_BPartner_Location_ID IS NULL AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:371)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.490===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.494-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.495-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:378)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.496===========> DB.executeUpdate: UPDATE I_Order SET I_IsImported='E', I_ErrorMsg=I_ErrorMsg||'ERR=No BP Location, ' WHERE C_BPartner_ID IS NOT NULL AND (BillTo_ID IS NULL OR C_BPartner_Location_ID IS NULL) AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:378)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.530===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.531-----------> NIImportOrder.doIt: No BP Location=-1 [186]
16:04:22.534-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.534-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:397)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.535===========> DB.executeUpdate: UPDATE I_Order SET C_Country_ID=(SELECT C_Country_ID FROM C_Country c WHERE I_Order.CountryCode=c.CountryCode AND c.AD_Client_ID IN (0, I_Order.AD_Client_ID)) WHERE C_BPartner_ID IS NULL AND C_Country_ID IS NULL AND CountryCode IS NOT NULL AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:397)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.558===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.563-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.564-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:404)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.565===========> DB.executeUpdate: UPDATE I_Order SET I_IsImported='E', I_ErrorMsg=I_ErrorMsg||'ERR=Invalid Country, ' WHERE C_BPartner_ID IS NULL AND C_Country_ID IS NULL AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:404)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.700===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.701-----------> NIImportOrder.doIt: Invalid Country=-1 [186]
16:04:22.704-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.705-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:415)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.706===========> DB.executeUpdate: UPDATE I_Order Set RegionName=(SELECT MAX(Name) FROM C_Region r WHERE r.IsDefault='Y' AND r.C_Country_ID=I_Order.C_Country_ID AND r.AD_Client_ID IN (0, I_Order.AD_Client_ID)) WHERE C_BPartner_ID IS NULL AND C_Region_ID IS NULL AND RegionName IS NULL AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:415)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.738===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.742-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.743-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:424)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.743===========> DB.executeUpdate: UPDATE I_Order Set C_Region_ID=(SELECT C_Region_ID FROM C_Region r WHERE r.Name=I_Order.RegionName AND r.C_Country_ID=I_Order.C_Country_ID AND r.AD_Client_ID IN (0, I_Order.AD_Client_ID)) WHERE C_BPartner_ID IS NULL AND C_Region_ID IS NULL AND RegionName IS NOT NULL AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:424)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.758===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.761-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.761-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:433)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.762===========> DB.executeUpdate: UPDATE I_Order SET I_IsImported='E', I_ErrorMsg=I_ErrorMsg||'ERR=Invalid Region, ' WHERE C_BPartner_ID IS NULL AND C_Region_ID IS NULL AND EXISTS (SELECT * FROM C_Country c WHERE c.C_Country_ID=I_Order.C_Country_ID AND c.HasRegion='Y') AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:433)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.771===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.772-----------> NIImportOrder.doIt: Invalid Region=-1 [186]
16:04:22.774-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.774-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:443)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.775===========> DB.executeUpdate: UPDATE I_Order SET M_Product_ID=(SELECT MAX(M_Product_ID) FROM M_Product p WHERE I_Order.ProductValue=p.Value AND I_Order.AD_Client_ID=p.AD_Client_ID) WHERE M_Product_ID IS NULL AND ProductValue IS NOT NULL AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:443)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.785===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.788-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.788-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:450)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.789===========> DB.executeUpdate: UPDATE I_Order SET M_Product_ID=(SELECT MAX(M_Product_ID) FROM M_Product p WHERE I_Order.UPC=p.UPC AND I_Order.AD_Client_ID=p.AD_Client_ID) WHERE M_Product_ID IS NULL AND UPC IS NOT NULL AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:450)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.798===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.800-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.800-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:457)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.802===========> DB.executeUpdate: UPDATE I_Order SET M_Product_ID=(SELECT MAX(M_Product_ID) FROM M_Product p WHERE I_Order.SKU=p.SKU AND I_Order.AD_Client_ID=p.AD_Client_ID) WHERE M_Product_ID IS NULL AND SKU IS NOT NULL AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:457)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.834===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.838-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.838-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:463)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.839===========> DB.executeUpdate: UPDATE I_Order SET I_IsImported='E', I_ErrorMsg=I_ErrorMsg||'ERR=Invalid Product, ' WHERE M_Product_ID IS NULL AND (ProductValue IS NOT NULL OR UPC IS NOT NULL OR SKU IS NOT NULL) AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:463)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.868===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.870-----------> NIImportOrder.doIt: Invalid Product=-1 [186]
16:04:22.871-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.872-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:472)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.872===========> DB.executeUpdate: UPDATE I_Order SET C_Charge_ID=(SELECT C_Charge_ID FROM C_Charge c WHERE I_Order.ChargeName=c.Name AND I_Order.AD_Client_ID=c.AD_Client_ID) WHERE C_Charge_ID IS NULL AND ChargeName IS NOT NULL AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:472)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.919===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.922-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.922-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:478)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.923===========> DB.executeUpdate: UPDATE I_Order SET I_IsImported='E', I_ErrorMsg=I_ErrorMsg||'ERR=Invalid Charge, ' WHERE C_Charge_ID IS NULL AND (ChargeName IS NOT NULL) AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:478)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.960===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:22.961-----------> NIImportOrder.doIt: Invalid Charge=-1 [186]
16:04:22.964-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:22.964-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:487)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:22.965===========> DB.executeUpdate: UPDATE I_Order SET I_IsImported='E', I_ErrorMsg=I_ErrorMsg||'ERR=Product and Charge, ' WHERE M_Product_ID IS NOT NULL AND C_Charge_ID IS NOT NULL AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:487)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:23.003===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:23.004-----------> NIImportOrder.doIt: Invalid Product and Charge exclusive=-1 [186]
16:04:23.006-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:23.006-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:497)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:23.007===========> DB.executeUpdate: UPDATE I_Order SET C_Tax_ID=(SELECT MAX(C_Tax_ID) FROM C_Tax t WHERE I_Order.TaxIndicator=t.TaxIndicator AND I_Order.AD_Client_ID=t.AD_Client_ID) WHERE C_Tax_ID IS NULL AND TaxIndicator IS NOT NULL AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:497)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:23.046===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:23.048-----------> NewPooledConnection.log: [c3p0] A PooledConnection that has already signalled a Connection error is still in use! [186]
16:04:23.049-----------> NewPooledConnection.log: [c3p0] Another error has occurred [ org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block ] which will not be reported to listeners! [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:503)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:23.050===========> DB.executeUpdate: UPDATE I_Order SET I_IsImported='E', I_ErrorMsg=I_ErrorMsg||'ERR=Invalid Tax, ' WHERE C_Tax_ID IS NULL AND TaxIndicator IS NOT NULL AND I_IsImported<>'Y' AND AD_Client_ID=1000000 [SvrProcess_7ce8d06b-5595-47c1-b361-f0da87a12f66] [186]
org.postgresql.util.PSQLException: ERROR: current transaction is aborted, commands ignored until end of transaction block; State=25P02; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:495)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:441)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:1449)
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.compiere.db.StatementProxy.invoke(StatementProxy.java:128)
	at com.sun.proxy.$Proxy3.executeUpdate(Unknown Source)
	at org.compiere.util.DB.executeUpdate(DB.java:1038)
	at org.compiere.util.DB.executeUpdate(DB.java:897)
	at org.compiere.util.DB.executeUpdate(DB.java:884)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:503)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)

16:04:23.092===========> DB.saveError: DBExecuteError - ERROR: current transaction is aborted, commands ignored until end of transaction block [186]
16:04:23.094-----------> NIImportOrder.doIt: Invalid Tax=-1 [186]
16:06:30.605===========> NIImportOrder.doIt: Order - SELECT * FROM I_Order WHERE I_IsImported='N' AND AD_Client_ID=1000000 ORDER BY C_BPartner_ID, BillTo_ID, C_BPartner_Location_ID, I_Order_ID [186]
java.lang.ArithmeticException: Non-terminating decimal expansion; no exact representable decimal result.
	at java.math.BigDecimal.divide(Unknown Source)
	at ni.idempiere.override.process.NIImportOrder.doIt(NIImportOrder.java:768)
	at org.compiere.process.SvrProcess.process(SvrProcess.java:198)
	at org.compiere.process.SvrProcess.startProcess(SvrProcess.java:144)
	at org.adempiere.util.ProcessUtil.startJavaProcess(ProcessUtil.java:172)
	at org.compiere.apps.AbstractProcessCtl.startProcess(AbstractProcessCtl.java:466)
	at org.compiere.apps.AbstractProcessCtl.run(AbstractProcessCtl.java:234)
	at org.adempiere.webui.apps.WProcessCtl.process(WProcessCtl.java:197)
	at org.adempiere.webui.apps.AbstractProcessDialog$ProcessDialogRunnable.doRun(AbstractProcessDialog.java:1060)
	at org.adempiere.util.ContextRunnable.run(ContextRunnable.java:38)
	at org.adempiere.webui.apps.DesktopRunnable.run(DesktopRunnable.java:40)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)

